src/styles/_catalog.scss

```diff
@@ -15,7 +15,7 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
-    transition: $translation-time cubic-bezier(.51,.11,0,.74);
+    transition: $translation-time;
```

Comment:
The same as in header comment - i think its better to write all properties here and conditionns too